Dockerfile.add-backend-versions

```
@@ -9,4 +9,6 @@ COPY run_server.sh run_server.sh
 
 # add a version link to the image description
 ARG version
-LABEL org.opencontainers.image.description "https://github.com/spiffe/tornjak/releases/tag/$version"
\ No newline at end of file
+LABEL org.opencontainers.image.description="Tornjak backend ($version): https://github.com/spiffe/tornjak/releases/tag/$version" \
```

This image creates Tornjak + arbitrary version of spire, defined by arg `version` to container build
```suggestion
LABEL org.opencontainers.image.description="Tornjak backend + SPIRE server ($version): https://github.com/spiffe/tornjak/releases/tag/$version" \
```